lib/app.js

```diff
@@ -55,8 +57,18 @@ app.get('/coffees/:id', async(req, res) => {
     const coffeeID = req.params.id;
 
     const data = await client.query(`
-      SELECT * FROM coffees 
-      WHERE coffees.id=$1
+    SELECT
+    coffees.id as id,
+    name_id,
+    name,
+    image,
+    description,
+    category,
+    price,
+    on_sale,
+    suppliers.supplier as supplier
+    from coffees left join suppliers on coffees.supplier_id=suppliers.id 
```

Comment:
great work with this lab, Kevin. Also love that you're utilizing the different joins that sql offers and experimenting with how to arrange your data, as well as having such a detailed object to give your app.